.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./node_modules/gts/"
```

Comment:
You can hopefully make this `"extends": "gts"`.

Reply:
Updated!